AnanasProject/AnanasProject/ViewControllers/SearchViewController/PdfViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PdfViewController.swift
+//  AnanasProject
+//
+//  Created by Анастасия Козлова on 12.10.2022.
+//
+
+import UIKit
+/// Открытие PDF файла
+final class PdfViewController: UIViewController {
+    
+    // MARK: - Constant
+    
+    private enum Constant {
+        static let pdfName = "SwiftRazrabPrilozhUIKit"
+        static let pdf = "pdf"
```

Comment:
```suggestion
        static let pdf = "pdf"
```
может назовешь формат или еще как то